README.md

```diff
@@ -25,6 +25,27 @@ Based on traditional API Gateways, it intercepts all incoming messages moving th
 
 It supports all the protocols AsyncAPI supports through [bindings](https://github.com/asyncapi/bindings).
 
-This **Event Gateway** is also compatible with the HTTP protocol, natively or trough an external provider like [Krakend.io](http://krakend.io).
+This **Event Gateway** is also compatible with the HTTP protocol, natively or through an external provider like [Krakend.io](http://krakend.io).
 
-AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community and also some officially maintained by the AsyncAPI org.
+AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community or officially maintained by the AsyncAPI org.
+
+# Goals
+
+## 1. Performance-first while ensuring message delivery
+The **Event Gateway** is a **stateless** solution that ensures messages are delivered as fast as possible using a minimal resource footprint.
```

Comment:
`minimal resource footprint` ❤️

Reply:
Go Green 💚 !

Now I'm thinking if `minimal runtime footprint` would fit better.

Reply:
tbh, I saw it more as a plus because of the cost, not going green 😄 `resource` is better I think